tools/apiview/parsers/cpp-api-parser/ParseAzureSdkCpp/ParseAzureSdkCpp.h

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is there a reason to keep this file anymore?  <iostream> already is included in the .cpp file